applications/crossbar/priv/couchdb/account/services.json

```diff
@@ -16,6 +24,10 @@
         "users": {
             "map": "function(doc) { if (doc.pvt_type != 'user' || doc.pvt_deleted || doc.enabled == false) return; emit(doc.priv_level || 'user', 1); }",
             "reduce": "_sum"
+        },
+        "voicemails": {
+            "map": "function(doc) { if (doc.pvt_type != 'vmbox' || doc.pvt_deleted || doc.enabled == false) return; emit('mailbox', 1); }",
```

Comment:
Why mailbox here? The other changes appear to follow the `pvt_type`. Also, why not a "generic" map of `if(deleted/not_enabled) return; emit(pvt_type, 1);`? Then query based on pvt_type and reduce to get the count?

Reply:
@jamesaimonetti I am not doing a generic pvt_type map to control churn on the services doc and only selectively quantity things that are billable for the cluster (as controlled in system_config).  It also makes more logical groupings for the items.  However, I have come around to using the pvt_type 'vmbox' rather than 'mailbox' so I will make that change now :)

Reply:
works for me :)